running-code/execute-script.md

```diff
@@ -0,0 +1,191 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(execute-script)=
+# Execute a Python script
+
+There are two primary ways to execute a Python script.
+
+You may already be familiar with the `python` command, and that it can take the name of a Python file and execute it
+
+```bash
+python my_program.py
+```
+
+When Python reads a file in this way, it executes all of the "top-level" commands that are not indented.
```

Comment:
Can you please clarify what you mean by executing all of the top-level commands that are not indented? I'm not sure what that means. Would that mean all functions? would it not executive code within a for loop?  
```suggestion
When Python reads and executes a file using this approach, it executes all of the "top-level" commands that are not indented.  This means that any code that is written directly in the file and that is not inside a function or class will run immediately when the file is executed. Functions and classes that are called in that file as commands will also run but the functions and classes themselves will not run. 
```

Jeremy, bare with me again. I think I understand your point here, but I'm getting stuck on the "top level" commands. see if you like my changes or want to adjust?!

Reply:
Hmm you have it right, I was trying to call out that function and class blocks are not executed (they are turned into code objects, which is what `def` and `class` _do_ when invoked, but the code within is not executed after being made a code object). Yes, loops and contexts would be executed. This point is perhaps too distracting from the main point of the lesson.